tests/test_acceptance.py

```diff
@@ -0,0 +1,41 @@
+import json
+
+import pytest
+
+from dcqc.parsers import CsvParser
+from dcqc.reports import JsonReport
+from dcqc.tests.test_abc import TestABC
+from dcqc.utils import open_parent_fs
+
+
+@pytest.mark.integration
+def test_json_report_generation(get_data):
+    # GIVEN a list of external tests to skip (to remain self-contained)
+    all_tests = TestABC.list_tests()
+    skipped_tests = [test.__name__ for test in all_tests if test.is_external_test]
+
+    # AND a subset of internal tests to be required (to verify suite status behavior)
+    required_tests = ["Md5ChecksumTest"]
```

Comment:
Would we want the specification of required tests to be part of `files.csv`?

Reply:
When I add the CLI, I will have to think of how I want to expose these settings. One drawback of including required or skipped tests in the CSV file is that if these are file-type specific, there will be a lot of redundant information. 

Another option is a YAML configuration file, which would allow for non-redundant filetype/test-specific configuration, but I need to think about it some more.

Reply:
You bring up good points.  I'm sure we will run into this in the near future, so we can discuss this then.  No need to decide right now.